src/Bicep.Cli.IntegrationTests/BuildCommandTests.cs

```diff
@@ -237,7 +267,7 @@ public async Task Build_Valid_SingleFile_WithDigestReference_ShouldSucceed()
         }
 
         [DataTestMethod]
-        [DynamicData(nameof(GetInvalidDataSets), DynamicDataSourceType.Method, DynamicDataDisplayNameDeclaringType = typeof(DataSet), DynamicDataDisplayName = nameof(DataSet.GetDisplayName))]
+        [DynamicData(nameof(GetParamData), DynamicDataSourceType.Method, DynamicDataDisplayNameDeclaringType = typeof(DataSet), DynamicDataDisplayName = nameof(DataSet.GetDisplayName))]
```

Comment:
This should be left as is. Did you mean to duplicate it and alter it to use `GetParamData`? (Perhaps that is the commented out code starting on line 305?)

Reply:
Yes, I will create InvalidParamDataSet probably for this